.docker/run.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+# name of image
+IMAGE=lab-website-renderer:latest
+
+# name of running container
+CONTAINER=lab-website-renderer
+
+# choose platform
+PLATFORM=""
+
+# default docker run command
+DOCKER_RUN="docker run"
+
+# fix interactive on windows
+if [[ $OSTYPE == msys* ]]; then
+    DOCKER_RUN="winpty docker run"
+fi
+
+# build docker image
+docker build ${PLATFORM} \
+    --tag ${IMAGE} \
+    --file ./.docker/Dockerfile .
+
+# run built docker image
+${DOCKER_RUN} ${PLATFORM} \
+    --name ${CONTAINER} \
+    --init \
+    --rm \
+    --interactive \
+    --tty \
+    --publish 4000:4000 \
+    --publish 35729:35729 \
+    --volume $PWD:/usr/src/app \
+    ${IMAGE} "$@"
```

Comment:
I see that you've replaced this file with a cleaned-up version. I assume this is so end-users have an easier time modifying it, and I also assume you didn't expect them to know much about bash, so you simplified it? In this particular case, removing the "&&" between `docker build` and `${DOCKER_RUN}` causes the container to run even if the build failed, which IMHO can result in confusing behavior when the user doesn't see their changes reflected in the running container.

If you prefer the build and the run to be two separate statements, you could also test the result of `docker build` and exit if it's not successful, e.g.:

```bash
# build docker image
docker build ${PLATFORM} \
    --tag ${IMAGE} \
    --file ./.docker/Dockerfile .

if [ $? -ne 0 ]; then
    echo "ERROR: Build failed, see above for details"
    exit 1
fi
```